l3kernel/l3msg.dtx

```diff
@@ -1736,30 +1736,30 @@
 % Error messages needed to actually implement the message system
 % itself.
 %    \begin{macrocode}
-\__kernel_msg_new:nnnn { kernel } { message-already-defined }
+\__kernel_msg_new:nnnn { msg } { message-already-defined }
   { Message~'#2'~for~module~'#1'~already~defined. }
```

Comment:
elsewhere we have shorter second args, ie only "already-defined" as "message" is already given by the first art "msg". There a few more cases below

Reply:
Shortened where possible